ports/raspberrypi/boards/archi/mpconfigboard.mk

```diff
@@ -0,0 +1,16 @@
+USB_VID = 0x2E8A
+USB_PID = 0x1043
+USB_PRODUCT = "ARCHI"
+USB_MANUFACTURER = "NEWSAN"
+
+CHIP_VARIANT = RP2040
+CHIP_FAMILY = rp2
+
+EXTERNAL_FLASH_DEVICES = "W25Q32JVxQ"
+
+CIRCUITPY__EVE = 1
+CIRCUITPY_PICODVI = 1
+
+FROZEN_MPY_DIRS += $(TOP)/frozen/Adafruit_CircuitPython_MPU6050
+FROZEN_MPY_DIRS += $(TOP)/frozen/Adafruit_CircuitPython_Pixel_Framebuf
+FROZEN_MPY_DIRS += $(TOP)/frozen/Adafruit_CircuitPython_Led_Animation
```

Comment:
You'll need to add these as frozen libraries to .gitmodules. Do a `git submodule add`.
Note that for `Adafruit_CircuitPython_LED_Animation`, the LED is all capital letters.
You had `NeoPixel` here before. Did you want to add it back? It's already in `frozen/`.

Reply:
I have already corrected it

Reply:
we take out the Adafruit_CircuitPython_Led_Animation

Reply:
Sure. It is still in `.gitmodules` in the latest push, though. Also did you want NeoPixel in or out?